src/FramedClient.ts

```diff
@@ -93,13 +93,14 @@ class FramedClient {
         })
     }
 
-    public async createObject(id: string, file: File, chatToken: OmnichannelChatToken | null = null): Promise<void> {
+    public async createObject(id: string, file: File, chatToken: OmnichannelChatToken | null = null, supportedImagesMimeTypes?: string[]): Promise<void> {

```

Comment:
Can you please set `supportedImagesMimeTypes` with default empty list to ensure backward compatibility?

`createObject(id: string, file: File, chatToken: OmnichannelChatToken | null = null, supportedImagesMimeTypes?: string[] = [])`

Updating AMSClient would break otherwise.

Reply:
sure, just fyi, this was tested and it didnt break, Ill make the changes

Reply:
done